src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    public static final String SPLITTER_REGEX = "\\W+";
+    public static final String BUY_STRING_VALUE = "buy";
+    public static final String SUPPLY_STRING_VALUE = "supply";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(readFromFile(fromFileName)));
+    }
+
+    private String readFromFile(String fromFileName) {
+        try {
+            return Files.readString(new File(fromFileName).toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't open file",e);
```

Comment:
You should use try-with-resources to ensure that all resources are closed properly after use. While Files.readString() doesn't require closing the resource, it's a good practice for consistency and future-proofing when the code might change.